tests/dat/actions/loggingTimeout.js

```diff
@@ -0,0 +1,39 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more contributor
```

Comment:
Is this file still needed given the switch to python?

Reply:
finally, there is only the nodejs:6 version of test action in the change set of the PR